rasa/train.py

```diff
@@ -12,13 +12,8 @@
 from rasa.nlu.model import Interpreter
 from rasa.utils.common import TempDirectoryPath
 
-from rasa.cli.utils import (
-    print_success,
-    print_warning,
-    print_error,
-    bcolors,
-    print_color,
-)
+from rasa.cli.utils import print_success, print_warning, print_error, print_color
```

Comment:
again, functions are imported by their names while the module itself should be imported

Reply:
I'll leave these as they are as this is going beyond the scope of this PR. Will change all imports from `rasa.shared` though which aren't imported by module.